packages/opentelemetry-instrumentation/src/instrumentation.ts

```diff
@@ -40,12 +39,8 @@
       enabled: true,
       ...config,
     };
-    this._logger = this._config.logger || new api.NoopLogger();
```

Comment:
Same here. I thought we were going to have a getter to `api.diag` here so that we would have a place to easily add a namespace in the future. For example in the future it could be populated with some logger created by `api.diag.withNamespace(instrumentationName)`